docs/statuses/es.md

```diff
@@ -2,7 +2,217 @@
 
 # es
 
-##### All missed: 0
+##### All missed: 39
 
-All lines are translated 😊
+
+### [cashier](https://github.com/Laravel-Lang/lang/blob/master/locales/es/packages/cashier.json)
+
+##### Missing: 1
+
+<table >
+<tr><td align="left" >
+Jane Doe
+</td>
+</tr>
+
+</table>
+
+
+[ [go back](../status.md) | [to top](#) ]
+
+### [fortify](https://github.com/Laravel-Lang/lang/blob/master/locales/es/packages/fortify.json)
+
+##### Missing: 1
+
+<table >
+<tr><td align="left" >
+The :attribute must be at least :length characters and contain at least one uppercase character
+</td>
+</tr>
+
+</table>
+
+
+[ [go back](../status.md) | [to top](#) ]
+
+### [jetstream](https://github.com/Laravel-Lang/lang/blob/master/locales/es/packages/jetstream.json)
+
+##### Missing: 20
+
+<table >
+<tr><td align="left" >
+Add a new team member to your team
+</td>
+</tr>
+<tr><td align="left" >
+Are you sure you want to delete this team? Once a team is deleted
```

Comment:
[This line for example](https://github.com/Laravel-Lang/lang/blob/master/locales/es/es.json#L56) is correctly translated but appears as missing and also appears incomplete. And there are many more cases

![image](https://user-images.githubusercontent.com/9275870/118430789-77704f80-b69a-11eb-865c-9fb8cc2ad651.png)

Reply:
None of the above translations correspond to valid translations

Reply:
Got it. I'll do it. Thank you!

Reply:
@luisprmat, hmm... Regularity is not working properly. I'll fix it.

![image](https://user-images.githubusercontent.com/10347617/118515782-5eb97580-b73e-11eb-90e7-797055a031e2.png)

![image](https://user-images.githubusercontent.com/10347617/118515817-6711b080-b73e-11eb-8d98-d4633ee4f3ff.png)

![image](https://user-images.githubusercontent.com/10347617/118515983-92949b00-b73e-11eb-8c8c-6182dd049bb6.png)

Reply:
I found the bug. In the near future I will fix it and inform you.